### PR TITLE
Issue Template

### PR DESCRIPTION
**Pull request by winapp2**
Issue Template

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+- [] PCSX2 Version (For example v.1.5.0):
+
+
+- [] PCSX2 options (For example Full boot, Clamping modes, DX or OpenGl?) :
```
Comment (ssakash) [inline ISSUE_TEMPLATE.md]: ``DX or OpenGl`` doesn't come under PCSX2 options. (they're renderers used by GSDX)

**Comment by willkuer** [comment]
Also as pcsx2 version and pcsx2 options are separate entries I would also suggest to split plugin versions and plugin settings.

I think most issues are related to graphics and a quick overview about gsdx settings would be good to have.

**Comment by winapp2** [comment]
I don't know. But I just wanted to wake up you guys to make this wonderful feature to be in github. Please can somebody pull a new commit with the right things.

**Comment by winapp2** [comment]
You know better then me what you'll need. Hope this feature comes.

**Comment by willkuer** [comment]
?

Was just a suggestion/discussion. Nobody was attacking you...

**Comment by winapp2** [comment]
@willkuer , you understand me wrong. I'm not a c++ programmer, but this feature needs to be in pcsx2. I knew, that this PR is wrong. I just wanted to wake up you guys to add this PR but with knowledge in c++, not like me a novice. I'm not attacked and attack other people.

**Comment by winapp2** [comment]
I'm not able to pull a request like this, because of my lack of experience. I don't know c++ or another language.

**Comment by FlatOutPS2** [comment]
There is no c++ required for this PR. He just asked for an overview of the video plugin (GSdx) options you can find in the corresponding PCSX2 menu.

**Comment by winapp2** [comment]
I really dont know anything about this here. I'm a noob in this :+1:

**Pull request closed without merging**